src/Widgets/EligibilityModal/index.tsx

```diff
@@ -28,17 +28,12 @@ const EligibilityModal: FunctionComponent<Props> = ({
   const ModalComponent = isBigScreen ? DesktopModal : MobileModal
   const eligiblePlans = eligibilityPlans.filter((plan) => plan.eligible)
   const currentPlan = eligiblePlans[currentPlanIndex]
-  const modalProps = {
-    eligibilityPlans: eligiblePlans,
-    currentPlanIndex,
-    setCurrentPlanIndex,
-    currentPlan,
-    status,
-  }
+
+  const isDeferred = currentPlan?.deferred_days > 0 || currentPlan?.deferred_months > 0
```

Comment:
The spec is:

> GIVEN That at least one deferred payment fee plan is available
> WHEN Displaying the widget modale
> THEN The title should be "Payez en plusieurs fois ou plus tard par carte bancaire avec Alma." (pay later is mentioned)

So, we should not evaluate if the currentPlan is deferred, but if in all our payment plans, one of them is deferred

Reply:
fixed :)